README.md

```diff
@@ -1196,8 +1196,8 @@ public class DependsOnSample {
 
 ## Samples
 
-Examples of use AEM Authoring Toolkit API and DependsOn library are presented in a separate [AAT Samples](./aem-authoring-toolkit-samples) module.
+Examples of use AEM Authoring Toolkit API and DependsOn library are presented in a separate [AAT Samples](samples) module.
 
-Run `mvn clean install -P install-samples` from the root folder of [AAT Samples](./aem-authoring-toolkit-samples) to __install__ a sample project.
+Run `mvn clean install -P install-samples` from the root folder of [AAT Samples](samples) to __install__ a sample project.
```

Comment:
`the` sample project, it is the only one)

Reply:
fixed